community/php7-mustache/APKBUILD

```diff
@@ -0,0 +1,55 @@
+# Contributor: Renoir Boulanger <hello@renoirboulanger.com>
+# Maintainer: John Boehr <jbboehr@gmail.com>
+pkgname=php7-mustache
+pkgver=0.7.2
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
+pkgdesc="PHP$_phpver extension: C++ implementation of Mustache"
+url="https://github.com/jbboehr/php-mustache"
+arch="all"
+license="MIT"
+# XXX: sudo apk add --no-cache --update --repository http://dl-cdn.alpinelinux.org/alpine/edge/community php7-dev php7-common
+depends="
+	php$_phpver-common
+	libmustache
+	"
+makedepends="
+	php$_phpver-dev
+	autoconf
+	automake
+	libtool
+	git
+	"
+subpackages=""
+
+source="
+	php-mustache-$pkgver.tar.gz::https://github.com/jbboehr/php-mustache/archive/v$pkgver.tar.gz
+	"
+
+builddir="$srcdir/php-mustache-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	git clone https://github.com/jbboehr/mustache-spec.git spec && \
+	cd spec && \
+	git reset --hard 72233f3ffda9e33915fd3022d0a9ebbcce265acd && \
+	cd ..
+	default_prepare || return 1
+}
+
+build() {
+	cd "$builddir"
+	phpize$_phpver
+	./configure --enable-mustache --prefix=/usr --with-php-config=php-config$_phpver
+	make
+}
+
+package() {
+	cd "$builddir"
+
+	# Gotta find how to fix this
```

Comment:
PHP modules use `INSTALL_ROOT` instead of `DESTDIR`, that's okay. Also don't know what's the problem here